diff --git a/dist/lib/nft.js b/dist/lib/nft.js
index 27147c99ad52f891317d9ab4e428e5f71c7ced65..b2f8f8e7fa3caee1beec25e5f71dc1d039df537c 100644
--- a/dist/lib/nft.js
+++ b/dist/lib/nft.js
@@ -17,7 +17,8 @@ async function copyDependenciesToFunction({
   const { nodeFileTrace } = await import("@vercel/nft");
   const result = await nodeFileTrace([entryPath], {
     base: fileURLToPath(base),
-    cache
+    cache,
+    ignore: (file) => file.match(/(?:^|\/|\\)ai(?:$|\/|\\)/) !== null && !file.includes('node_modules')
   });
   for (const error of result.warnings) {
     if (error.message.startsWith("Failed to resolve dependency")) {
