diff --git a/dist/index.js b/dist/index.js
index 7f88cb87bf3268ced38aa1c4c7bc6546757a4379..e04347167e06b239eb0c8f49538f669072cf13e3 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -236,7 +236,9 @@ function vercelAdapter({
           mkdirSync(new URL("./.vercel/output/static/", _config.root), {
             recursive: true
           });
-          mkdirSync(new URL("./.vercel/output/server/", _config.root));
+          mkdirSync(new URL("./.vercel/output/server/", _config.root), {
+            recursive: true
+          });
           if (_buildOutput !== "static") {
             cpSync(_config.build.server, new URL("./.vercel/output/_functions/", _config.root), {
               recursive: true
diff --git a/dist/lib/nft.js b/dist/lib/nft.js
index 27147c99ad52f891317d9ab4e428e5f71c7ced65..b2f8f8e7fa3caee1beec25e5f71dc1d039df537c 100644
--- a/dist/lib/nft.js
+++ b/dist/lib/nft.js
@@ -17,7 +17,8 @@ async function copyDependenciesToFunction({
   const { nodeFileTrace } = await import("@vercel/nft");
   const result = await nodeFileTrace([entryPath], {
     base: fileURLToPath(base),
-    cache
+    cache,
+    ignore: (file) => file.match(/(?:^|\/|\\)ai(?:$|\/|\\)/) !== null && !file.includes('node_modules')
   });
   for (const error of result.warnings) {
     if (error.message.startsWith("Failed to resolve dependency")) {
