diff --git a/dist/lib/nft.js b/dist/lib/nft.js
index 0000000..1111111 100644
--- a/dist/lib/nft.js
+++ b/dist/lib/nft.js
@@ -17,7 +17,8 @@ async function copyDependenciesToFunction({
   const { nodeFileTrace } = await import("@vercel/nft");
   const result = await nodeFileTrace([entryPath], {
     base: fileURLToPath(base),
-    cache
+    cache,
+    ignore: (file) => file.match(/(?:^|\/|\\)ai(?:$|\/|\\)/) !== null && !file.includes('node_modules')
   });
   for (const error of result.warnings) {
     if (error.message.startsWith("Failed to resolve dependency")) {
