diff --git a/dist/core/build/plugins/plugin-manifest.js b/dist/core/build/plugins/plugin-manifest.js
index 8f5346f8caf8e0b5a990665620a1e3985e6dfd08..125c99ec4dd73faa2004b87683955e595361ae09 100644
--- a/dist/core/build/plugins/plugin-manifest.js
+++ b/dist/core/build/plugins/plugin-manifest.js
@@ -107,7 +107,8 @@ async function buildManifest(opts, internals, staticFiles, encodedKey) {
     ])
   );
   if (settings.scripts.some((script) => script.stage === "page")) {
-    staticFiles.push(rawEntryModules[PAGE_SCRIPT_ID]);
+    const src = rawEntryModules[PAGE_SCRIPT_ID];
+    if (src) staticFiles.push(src);
   }
   const prefixAssetPath = (pth) => {
     let result = "";
@@ -140,10 +141,12 @@ async function buildManifest(opts, internals, staticFiles, encodedKey) {
     const scripts = [];
     if (settings.scripts.some((script) => script.stage === "page")) {
       const src = rawEntryModules[PAGE_SCRIPT_ID];
-      scripts.push({
-        type: "external",
-        value: appendAssetQuery(src)
-      });
+      if (src) {
+        scripts.push({
+          type: "external",
+          value: appendAssetQuery(src)
+        });
+      }
     }
     const links = [];
     const styles = pageData.styles.sort(cssOrder).map(({ sheet }) => sheet).map((s) => s.type === "external" ? { ...s, src: appendAssetQuery(s.src) } : s).reduce(mergeInlineCss, []);
